lib/heimdall_tools/nessus_mapper.rb

```diff
@@ -2,10 +2,12 @@
 require 'csv'
 require 'heimdall_tools/hdf'
 require 'utilities/xml_to_hash'
+require 'nokogiri'
 
 RESOURCE_DIR = Pathname.new(__FILE__).join('../../data')
```

Comment:
There was some conversation with @rbclark and @Bialogs on the right way to do this at the gem level to make sure we are always referencing the data files in the right location and that the data in the compiled gem was not actually an xls but a proper ruby data object. Does this incorporate this?

Reply:
Yes Pathname gets us the absolute path and is suitable to work from a gem.